nbdime/utils.py

```diff
@@ -157,6 +182,21 @@ def locate_gitattributes(global_=False):
                 gitattributes = os.path.expandvars('$XDG_CONFIG_HOME/git/attributes')
             else:
                 gitattributes = os.path.expanduser('~/.config/git/attributes')
+    elif scope == 'system':
+        # git docs: "Attributes for all users on a system should be placed in
+        # the $(prefix)/etc/gitattributes file". Our job is then to check for
+        # $(prefix) value.
+        try:
+            with set_env(GIT_EDITOR='echo'):
+                bpath = check_output(['git', 'config', '--system', '-e'])
```

Comment:
You can pass an `env=` argument to check_output. You don't need to modify the current process's env to do this.